scalafix-tests/input/src/main/scala/test/NoInferConfig.scala

```diff
@@ -0,0 +1,16 @@
+/*
+rule = NoInfer
+NoInfer.symbols = [
+  "_root_.scala.Predef.any2stringadd(Ljava/lang/Object;)Ljava/lang/Object;."
```

Comment:
is there any better way I could use to express this symbol ? I couldn't make it work without the full name. Thanks

Reply:
I think the normalized version should work. It should be `_root_.scala.Predef.any2stringadd.` (notice the final `.`)

Reply:
althoug, now that I see the implementation, it would require `SymbolMatcher.normalized` instead of `SymbolMatcher.exact`, so no, you have to use the exact version.